master/buildbot/status/persistent_queue.py

```diff
@@ -308,7 +310,7 @@ def insertBackChunk(self, chunk):
         if excess > 0:
             chunk2 = []
             for i in range(excess):
-                chunk2.append(self.primaryQueue.items().pop())
+                chunk2.append(list(iteritems(self.primaryQueue)).pop())
```

Comment:
This one isnt right ... that queue is IQueue and has an 'items' method. I think you leave this one.